gallery/src/data-visualization/ReactVis.scss

```diff
@@ -0,0 +1,33 @@
+@import "~react-vis/dist/style";
+
+.nx-donut-hint {
+  background-color: black;
+  border-radius: 5px;
+  color: white;
+  font-size: 12px;
+  padding: 8px;
+}
+
+.nx-graph-shape-fill-dark {
```

Comment:
After looking at https://github.com/uber/react-vis/blob/master/docs/colors.md, it looks like coloring via CSS isn't really supported.  I think what we need to do is create a `colorRange` based on the RSC selectable colors (or maybe two, one for the light selectable colors and one for the dark) and document how to use that.